tests/ops/functions/test_bind_new_parameters.py

```diff
@@ -184,28 +185,32 @@ def test_controlled_sequence():
     qml.assert_equal(new_op.base, qml.RX(0.5, wires=3))
 
 
-with qml.operation.disable_new_opmath_cm():
-    TEST_BIND_LEGACY_HAMILTONIAN = [
-        (
-            qml.ops.Hamiltonian(
-                [1.1, 2.1, 3.1],
-                [Tensor(qml.PauliZ(0), qml.PauliX(1)), qml.Hadamard(1), qml.PauliY(0)],
-            ),
-            [1.2, 2.2, 3.2],
-            qml.ops.Hamiltonian(
+with qml.operation.disable_new_opmath_cm(warn=False):
+    with warnings.catch_warnings():
+        warnings.filterwarnings(
+            "ignore", "qml.ops.Hamiltonian uses", qml.PennyLaneDeprecationWarning
+        )
+        TEST_BIND_LEGACY_HAMILTONIAN = [
```

Comment:
Why do we need to create this in a context manager instead of directly in the parametrization?

Reply:
Thanks for catching this. I feel like this test should be tested in normal CI as well, so I've removed the `"legacy_opmath_only` fixture and removed the `disable_new_opmath_cm` context manager. Since we're explicitly using `Tensor` and `qml.ops.Hamiltonian`, I'll leave the warnings filter.